### PR TITLE
add build/README

### PR DESCRIPTION
**Pull request by CharlesLvw**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
English version for Chinese doc: https://github.com/kubeedge/kubeedge/blob/master/build/README_zh.md

please sign off the commit to pass the DCO :)
ref: https://github.com/kubeedge/kubeedge/pull/3379/checks?check_run_id=4321914926

File: build/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- cloud
+- crds
+- crd-samples
+- deployment.yaml
+- edge
+- node.json
+- tools
+
+## cloud
+Relevant steps and instructions for edegcontroller deployment with kubernetes deployment
```
Comment (fisherxu) [inline build/README.md:53]:
```suggestion
Relevant steps and instructions for cloudcore deployment with kubernetes deployment

```

File: build/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+## crds
+
+## crd-samples
+
+## edge
+Relevant steps and instructions for edgecore deployment with kubernetes deployment
```
Comment (fisherxu) [inline build/README.md:61]:
```suggestion
Relevant steps and instructions for edgecore deployment with kubernetes deployment (Only for test)

```

**Review by fisherxu — COMMENTED** [review]
Could you also please update the chinese doc? Thanks.

**Comment by CharlesLvw** [comment]
> 

done

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**